best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
+(e.g. `CreatePurchase`, `UpdateCreditCard`).
+* Expose one public method on services that does the work
+(i.e. `run`, `call`, or `perform`).
```

Comment:
My intent was to narrow down to a handful of method names. `call` might be the best because that is also what Ruby lambda's use, thus a service can be stubbed with a lambda. I've used `run` often in the past -- as in running a service.